weblate/templates/translate.html

```diff
@@ -318,31 +318,31 @@ <h4 class="panel-title">
     </tbody>
     {% endif %}
     {% if others.matching %}
-    <tr><th colspan="5" class="warning">{% blocktrans count count=others.matching|length %}The following string has the same context and source.{% plural %}The following strings have the same context and source.{% endblocktrans %}</th></tr>
+    <tr><th colspan="5" class="warning">{% blocktrans count count=others.matching|length %}This string has the same context and source.{% plural %}These strings have the same context and source.{% endblocktrans %}</th></tr>
     <tbody>
     {% for item in others.matching %}
     {% include "trans/other-row.html" %}
     {% endfor %}
     </tbody>
     {% endif %}
     {% if others.source %}
-    <tr><th colspan="5" class="warning">{% blocktrans count count=others.source|length %}The following string has different context, but the same source.{% plural %}The following strings have different contexts, but the same source.{% endblocktrans %}</th></tr>
+    <tr><th colspan="5" class="warning">{% blocktrans count count=others.source|length %}This string has a different context, but the same source.{% plural %}These strings have differing context, but the same source.{% endblocktrans %}</th></tr>
     <tbody>
     {% for item in others.source %}
     {% include "trans/other-row.html" %}
     {% endfor %}
     </tbody>
     {% endif %}
     {% if others.context %}
-    <tr><th colspan="5" class="warning">{% blocktrans count count=others.context|length %}The following string has a different source, but the same context.{% plural %}The following strings have different sources, but the same context.{% endblocktrans %}</th></tr>
+    <tr><th colspan="5" class="warning">{% blocktrans count count=others.context|length %}This string has a different source, but the same context.{% plural %}These strings have differing sources, but the same context.{% endblocktrans %}</th></tr>
```

Comment:
("differing" to make it clear it is different to the original string, as opposed to just all different between each-other.)

Reply:
Shouldn't the singular have differing as well?

Reply:
Not how I think of it. Might have to consult someone.